packages/sit-onyx/src/components/OnyxTable/OnyxTable.vue

```diff
@@ -61,6 +71,13 @@ const style = computed(() => ({
   "--onyx-table-observed-height": `${height.value}px`,
   "--onyx-table-observed-width": `${width.value}px`,
 }));
+
+const columnCount = computed(() => {
```

Comment:
Hmm I am not sure if we should iterate the vnodes here...for the data grid we know the columns because they are passed as prop.

For the OnyxTable we might consider requiring the user to pass the column count as well. We can talk about it in the daily